.github/dependabot.yml

```diff
@@ -3,5 +3,6 @@ updates:
 - package-ecosystem: npm
   directory: "/"
   schedule:
-    interval: monthly
+    interval: weekly
```

Comment:
nit: can we keep the config args consistent re: having quotes or not? example shows both w/ quotes: https://help.github.com/en/github/administering-a-repository/configuration-options-for-dependency-updates#scheduleday